ui/DesignSystem/Button.svelte

```diff
@@ -96,33 +88,17 @@
   }
 
   .primary:hover {
-    background-color: var(--color-primary-level-2);
-    border-color: var(--color-primary-level-2);
+    border-color: var(--color-primary);
     color: var(--color-background);
   }
 
-  .primary:active {
-    background-color: var(--color-primary-level-2);
-    border-color: var(--color-primary-level-2);
-    color: var(--color-primary-level-1);
-  }
-
-  .primary:active :global(svg) {
-    fill: var(--color-primary-level-1);
-  }
-
   .primary.disabled {
-    background-color: var(--color-primary-level-1);
-    border-color: var(--color-primary-level-1);
-    color: var(--color-background);
+    opacity: 0.25;
     cursor: not-allowed;
   }
 
   .primary.disabled:hover {
-    background-color: var(--color-primary-level-1);
-    box-shadow: 0 0 0 0;
-    border-color: var(--color-primary-level-1);
-    color: var(--color-background);
+    transform: scale(1);
```

Comment:
Why do we need this here and on the other selectors? For me, everything seems to work without it.